src/Handler/Command.hs

```diff
@@ -56,6 +59,9 @@ patchCommandR token = do
 getCommandR :: Token -> Handler TypedContent
 getCommandR token = do
     command <- unsafeRunStorage $ get404 token
+    mcontent <- if commandArchived command
+        then Just <$> getOutput token sinkLbs
```

Comment:
not that it's any better but for my own curiosity could you use `pure` here like so:
`then pure =<< getOutput token sinkLbs` or not because it doesn't know what `m` you want?